mono/mini/debugger-agent.c

```diff
@@ -1091,12 +1091,18 @@ mono_debugger_get_generate_debug_info ()
 	return disable_optimizations;
 }
 
+// Declare transport_close2 method
+static void transport_close2(void);
```

Comment:
I think it would be nice if you put this forward declaration next to the others ones at the top of file and also follow the convention of having a space between the function name and the parameter :-) 

This will make it easier to upstream the changes.